api/messages.proto

```diff
@@ -239,9 +236,10 @@ message Match {
   reserved 5, 6;
 }
 
+// Represents a backfill entity which is used to fill partially full matches.
+// 
```

Comment:
You removed this newline between the comment and the beta feature warning elsewhere, why add it here?

Reply:
My initial intention was to keep a separate line between object description and the beta feature warning but i realized this only works at top level. If i put a separate line in an inner field, the markdown generated has tables broken visually